src/components/EditableParagraph.jsx

```diff
@@ -10,6 +10,7 @@ const EditablePagrapraph = (props) => {
 
   const paragraphEditor = useRef();
 
+  const [text, setText] = useState(value);
```

Comment:
It seems the `text` `useState` is being used as a local version of `value`, so why not call it `localValue` or something like that to make that clearer? It is better to use one cohesive name.

If you feel `value` is not good I would change `value` to `text` and `text` to `localText`,

Reply:
I think this is already pretty clear. We usually pass in `value` to an input component. However, inside the component I used `text` because I'm not really using a real `input` component, but a div with `contenteditable`, which should display `text` inside it, not `value`.

Reply:
But `text` is the same as `value` but internal, so why use a completely different name?

I think it is clear right now because there isn't much code, the problem is more noticeable in larger files, but it is still a good practice.

Other naming ideas could be `inputText` `text` or `inputValue` and `value`, you can rename those on destructuring  the `props` so that from outside we only see `value` if you want.

Reply:
Hmm... ok, that's a good argument. I'll change the name,